migrations/20180921123650_easyengine_test.php

```diff
@@ -7,7 +7,7 @@
  *
  * @package EE\Migration
  */
-class Blank extends \EE\Migration\Base {
+class Test extends \EE\Migration\Base {
```

Comment:
> we just need a directory with empty file in it(like .gitkeep).
`.gitkeep` will work for `git`, but the I don't think it will work for [Symphony Finder](https://symfony.com/doc/current/components/finder.html).

Reply:
> we just need a directory with empty file in it(like .gitkeep).

`.gitkeep` will work for `git`, but the I don't think it will work for [Symphony Finder](https://symfony.com/doc/current/components/finder.html).

Reply:
We can intentionally add that file in first few releases of phar until we have first migration and after that we can remove that file.